pkg/master/master.go

```diff
@@ -585,11 +589,20 @@ func (m *Master) initV1ResourcesStorage(c *Config) {
 }
 
 // NewBootstrapController returns a controller for watching the core capabilities of the master.
-func (m *Master) NewBootstrapController() *Controller {
+func (m *Master) NewBootstrapController(c *Config) *Controller {
+	var masterLeases leases.Leases
+	if c.EnableLeases {
+		leaseStorage, err := c.StorageFactory.New(api.Resource("apiServerIPInfo"))
```

Comment:
since we no longer use a new API type, is this resource name string still correct?